events/static/js/app/AddCompanyWindow.js

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import AutoComplete from 'material-ui/AutoComplete';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import CompanyTextField from './CompanyTextField';
+import axios from 'axios';
+export default class AddCompanyWindow extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
@VKotsovskyi, `getInitialState` was used in ES5. We are using ES6, so `this.state= {...}` is fine.